ConcordiumWallet/Model/AppSettingsResponse.swift

```diff
@@ -0,0 +1,20 @@
+//
+//  AppSettingsResponse.swift
+//  ConcordiumWallet
+//
+//  Created by Lars Christensen on 16/05/2022.
+//  Copyright © 2022 concordium. All rights reserved.
+//
+
+import Foundation
+
+struct AppSettingsResponse: Codable {
+    let status: String
+    let url: String?
+}
```

Comment:
```suggestion
enum AppSettingsStatus: String, Codable {
    case ok
    case warning
    case needsUpdate
}

struct AppSettingsResponse: Codable {
    let status: AppSettingsStatus
    let url: URL?
}
```

I would probably create an enum for the status property and have the url property use the Url type. That way it is automatically handled by codable and we can avoid a nasty unwrapping later in the code.

Reply:
Thanks, I have changed it.